website/docs/plugin/framework/migrating/testing.mdx

```diff
@@ -0,0 +1,170 @@
+---
+page_title: 'Testing Migration: Migrating from SDKv2 to the Framework'
+description: >-
+  Write tests that verify that switching from SDKv2 to the Framework does not affect provider behavior.
+---
+
+# Testing
+
+During migration, you should [write tests](#testing-migration) to verify that the behaviour of your provider has not
+been altered by the migration itself. You will also need to [update](#provider-factories) your tests too.
+
+## Testing Migration
+
+During migration, we recommend writing tests to verify that switching from SDKv2 to the Framework has not affected your
+provider's behavior. These tests use identical configuration. The first test step applies a plan and generates state
+with the SDKv2 version of the provider. The second test step generates a plan with the Framework version of the provider
+and verifies that the plan is a no-op indicating that migrating to the framework has not altered behaviour.
+
+Use the `ExternalProviders` field within a `resource.TestStep` to specify the configuration of a specific provider to
+use during each test step. You can specify a version of the provider built on SDKv2 during the first test step, and
+then you can use the version of the provider built on the Framework in subsequent test step(s) to verify that Terraform
+CLI does not detect any planned changes.
+
+You must also update the [provider factories](/plugin/framework/migrating/testing/provider-factories) to use
+the Framework.
```

Comment:
Did the `testing/provider-factories` page not get migrated? I don't see it in the PR. (Also should `testing.mdx` be `testing/index.mdx`? @laurapacilio

Reply:
`Provider Factories` are included within the `testing.mdx` page. I've update the link. 

Let me know about `testing.mdx => `/testing/index.mdx`.